scripts/add_electricity.py

```diff
@@ -801,6 +802,55 @@ def estimate_renewable_capacities_irena(
             ] * float(p_nom_max)
 
 
+def attach_enhanced_geothermal(n):
+
+    egs_potential = pd.read_csv(snakemake.input["egs_potentials"], index_col=[0, 1])
+
+    idx = pd.IndexSlice
+
+    n.add(
+        "Bus",
+        "EGS",
+        carrier="geothermal heat",
+        unit="MWh_th",
+    )
+
+    n.add(
+        "Generator",
+        "EGS",
+        bus="EGS",
+        carrier="geothermal heat",
+        p_nom_extendable=True,
+    )
+
+    for bus in tqdm(
+        egs_potential.index.get_level_values(0).unique(),
+        desc="Adding enhanced geothermal",
+        ):
+
+        ss = egs_potential.loc[idx[bus, :]]
+
+        # Loop over supply‐curve point (i.e. each potential) for this region
+        for i, (cost_index, row) in enumerate(ss.iterrows()):
+            capacity = row["available_capacity[MW]"]
+            # Convert the cost index from $/kW to $/MW and annuitize the capex
+            capex_value = float(cost_index) * 1000  
+            ann_capex = capex_value * 0.07 / (1 - (1 + 0.07) ** (-25))
```

Comment:
the discount factor shall be taken from the config file. there is already the parameter
Rather than converting the value here, why don't we create the input data of ss converted into its annuity? Still the value shall be loaded from the config file